spec/test-utils/test-data/generate-test-data.py

```diff
@@ -265,6 +280,35 @@ def sign_json(json_object: dict, private_key: ed25519.Ed25519PrivateKey) -> str:
 
     return signature_base64
 
+def build_exported_megolm_key() -> dict:
+    index = int(0)
+    private_key = ed25519.Ed25519PrivateKey.generate()
+    # Just use radom bytes for the ratchet parts
+    ratchet = randbytes(32 * 4)
```

Comment:
I'd prefer to avoid randomness in this script, so that when we re-run it, only important things change.

Reply:
Added a static seed to have stable random at each generation

Reply:
can't we just hardcode the strings, rather than having to seed the random generator and call randbytes?

Reply:
Looks to me that it has best of the two words, it's stable and later on I can add more keys easilly test key upgrade and more